packages/swap/src/routes/v2/swap.ts

```diff
@@ -79,10 +79,6 @@ router.get(
         effectiveBoostFeeBps = 0;
       }
     }
-    const { srcAsset, destAsset } = {
```

Comment:
only used these variable for estimating the swap duration. also we already had `internalSrcAsset` and `internalDestAsset`